SharedPackages/DataBrokerProtectionCore/Package.swift

```diff
@@ -44,7 +44,7 @@ let package = Package(
                 .product(name: "Configuration", package: "BrowserServicesKit"),
                 .product(name: "Persistence", package: "BrowserServicesKit"),
             ],
-            resources: [.copy("Resources")],
+            resources: [.copy("BundleResources")],
```

Comment:
A weird build bug makes this necessary. The alternative was using .process instead, but we decided to stick to copy and rename it since the reason it was .copy in the first place was in the past we've relied on the file structure of the resources folder to identify fake brokers

Reply:
Yeah shame this is needed, but I think this approach seems the best of the bunch for now.